lib/components/AlbumScreen/song_list_tile.dart

```diff
@@ -437,3 +382,155 @@ class _SongListTileState extends ConsumerState<SongListTile>
+    this.isInPlaylist = false,
+    this.onRemoveFromList,
+    this.themeCallback,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+
+    jellyfin_models.BaseItemDto baseItem = item;
+
+    final cardBackground = Theme.of(context).brightness == Brightness.dark
+        ? const Color.fromRGBO(255, 255, 255, 0.075)
+        : const Color.fromRGBO(255, 255, 255, 0.125);
+
+    Widget listTile = Opacity(
+      opacity: isPlayable ? 1.0 : 0.5,
+      child: Card(
+        color: cardBackground,
+        elevation: 0,
+        margin:
+            const EdgeInsets.only(left: 6.0, right: 6.0, top: 6.0),
+        clipBehavior: Clip.antiAlias,
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(8.0),
+        ),
+        child: ListTile(
+          visualDensity: const VisualDensity(
+            horizontal: 0.0,
+            vertical: 0.0,
+          ),
+          minVerticalPadding: 0.0,
+          horizontalTitleGap: 8.0,
+          contentPadding: const EdgeInsets.symmetric(vertical: 0.0, horizontal: 0.0),
+          tileColor: isCurrentTrack
+              ? Theme.of(context).colorScheme.primary.withOpacity(0.1)
+              : const Color.fromRGBO(0, 0, 0, 0.035),
+          leading: Stack(
+            children: [
+              AlbumImage(
+                item: baseItem,
+                borderRadius: BorderRadius.zero,
+                themeCallback: themeCallback,
+              ),
+              if (isCurrentTrack)
+                SizedBox.square(
+                  dimension: 56,
+                  child: Container(
+                    // color: Theme.of(context).colorScheme.primary.withOpacity(0.35),
+                    color: Theme.of(context).brightness == Brightness.dark ? Colors.black.withOpacity(0.35) : Colors.white.withOpacity(0.35),
+                    child: MiniMusicVisualizer(
+                      color: Theme.of(context).colorScheme.secondary,
+                      animate: true,
+                    ),
+                  ),
+                ),
+            ],
+          ),
+          title: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Padding(
+                padding: const EdgeInsets.only(top: 1.0),
+                child: Text(
+                  baseItem.name ?? AppLocalizations.of(context)!.unknownName,
+                  style: isCurrentTrack
+                      ? TextStyle(
+                          color:
+                              Theme.of(context).colorScheme.secondary,
+                          fontSize: 16,
+                          fontWeight: FontWeight.w400,
+                          height: 1.0)
+                      : const TextStyle(
+                          fontSize: 16,
+                          height: 1.0),
+                  overflow: TextOverflow.ellipsis,
+                  maxLines: 2,
```

Comment:
Would `maxLines: 1` maybe look better here? Songs with long titles look a bit "compressed" right now (or maybe the line height is wrong? I prefer to use provided fonts like "headline" instead of defining them myself. I'm pretty sure you can dig around and find what `ListTile` uses)

![Screenshot of Infinity on High, which includes a lot of long titles](https://github.com/user-attachments/assets/93dbb2ea-55d9-4f7f-bf7d-50d33070e5ba)

Reply:
I was aiming to fit two rows onto a tile instead of just one. I agree that it looks off. I'll think about it...